javascript/examples/generate-audio-multiturn.ts

```diff
@@ -0,0 +1,80 @@
+import audioContext from "audio-context";
+import decodeAudio from "audio-decode";
+import play from "audio-play";
+import { openaiAudioModel } from "./model.js";
+
+const response = await openaiAudioModel.generate({
+  extra: {
+    audio: {
+      voice: "alloy",
+      format: "mp3",
+    },
+  },
+  modalities: ["text", "audio"],
+  messages: [
+    {
+      role: "user",
+      content: [
+        {
+          type: "text",
+          text: "Is a golden retriever a good family dog?",
+        },
+      ],
+    },
+  ],
+});
+
+console.dir(response, { depth: null });
```

Comment:
We can remove the debug log

Reply:
This is just example code. We can keep it.